sdks/python/apache_beam/examples/flink/flink_streaming_impulse.py

```diff
@@ -57,15 +58,24 @@ def run(argv=None):
     args.extend(argv)
 
   parser = argparse.ArgumentParser()
-  _, pipeline_args = parser.parse_known_args(args)
+  parser.add_argument('--count',
+                      dest='message_count',
+                      default=10000,
```

Comment:
Should the default perhaps be maxint, i.e. never quit?

Reply:
+1 If I remember correctly, `0` is used to emit forever.